cammac60/MacRae-Photo
# Add image

### What does this PR do? 

- Adds an additional photo.

----------

### How to manually test: 

- `git fetch`
- `git checkout popup-media-query`
- `npm start`
- `npm test` to ensure all tests are passing.
- Open the images page and click through the photos to make sure no errors occur.

---------

### Tickets: 

- None

----------

### Notes: 

 None
